docs/modules/ROOT/pages/how-tos/upgrade-mimir-v1-v2.adoc

```diff
@@ -83,6 +86,8 @@ yq eval -i ".parameters.${MIMIR_PARAM_KEY}.helm_values.alertmanager.zoneAwareRep
 
 yq eval -i ".parameters.${MIMIR_PARAM_KEY}.helm_values.rollout_operator.enabled = true" ${CLUSTER_ID}.yml
 
+git commit -am'Migrate alertmanager - step 1' && git push origin master
```

Comment:
Probably makes sense to note which alertmanager we're migrating here (the same applies for most of the other commit messages).

```suggestion
git commit -am "Migrate $MIMIR_INSTANCE alertmanager - step 1" && git push origin master
```